docs/managing-policy.md

```diff
@@ -1,28 +1,29 @@
 
-![Sample policy with rules set to send notifcations and block builds](img/policy-sample.png "Policy to block and notify when results are found.")
+![Sample policy with rules set to send notifcations and post PR comments](img/policy-sample.png "Policy to notify when results are found but not block the CI build")
 
 [TOC]
 
-# Creating policy
+# Creating a policy
 
-To create a policy, visit [Manage > Policies](https://semgrep.dev/manage/policy) and select “New Policy.”
+To create a policy, visit [Manage > Policies](https://semgrep.dev/manage/policy) and select “Create New Policy.” To copy the contents of one policy into another, navigate to the existing policy, click "Copy", and then enter the name you would like to give to the new policy.
 
-Policies are often broken down by problem area (e.g., `xss`), application type (e.g., `prod-python-backend`), or blocking status (e.g., `notify-only`). There is no right way to group rules, and it changes team-to-team and organization-to-organization.
+Policies are often broken down by problem area (e.g., `xss`), application type (e.g., `prod-python-backend`), or blocking status (e.g., `notify-only`). There is no right way to group rules, and what makes the most sense will vary by team and organization.
 
-# Editing policy
+# Editing a policy
 
-Any rule, ruleset, or pattern can be added to a policy. Look for the “Add to Policy” button.
+Any rule, ruleset, or pattern can be added to a policy. Look for the “Add to Policy” button when exploring [pre-written rules and rulesets](https://semgrep.dev/explore) or customize your policies even further by adding [rules you write yourself](https://semgrep.live) in our online playground.
 
 ![A ruleset with an "Add to Policy" button visible](img/ruleset.png "A ruleset showing 'Add to Policy'")
 
-To remove or edit the settings for a rule, ruleset, or pattern, go to [Manage > Policies](https://semgrep.dev/manage/policy) and select the relevant policy. You can then remove the item using the deletion `x` or change its notification settings through the ”Send Notification” and “Block CI” checkboxes.
+To edit the settings for a policy, go to [Manage > Policies](https://semgrep.dev/manage/policy) and select the relevant policy from the list. Add or rename integrations using the "Integrations" tab in the left-hand menu, and then add as many of these integrations as you want to each of your policies. You can also check or uncheck the boxes to post PR comments (which also requires a GitHub Token) or to block the build on findings. Don't forget to click save when you are finished editing!
```

Comment:
Perhaps link from “…also requires a GitHub token” to a help section about that? Somewhere like `integrations/#automatic-pr-comments-beta`?